tools/luacheck_config.lua

```diff
@@ -430,6 +430,16 @@ stds.nodemcu_libs = {
             MEDIUM_PRIORITY = empty,
             HIGH_PRIORITY = empty
           }
+        },
+        LFS = {
+          read_only = true, 
+          fields = {
```

Comment:
Not really a question for this PR, but since you're in the neighborhood: should we suggest to advanced developers how to add entries here?  People using their own LFS might want their code to still be `luacheck`-clean?

Reply:
@nwf I think it would be a good idea to make some kind of documentation on how to use `luacheck` on NodeMCU code. I've written `luacheck_config_helper.lua` script when I was adding `luacheck` support but I haven't documented it anywhere besides some comments in the script itself. Should this be added to GitHub Wiki,  Extension Developer FAQ or somewhere else?

Reply:
> I think it would be a good idea to make some kind of documentation on how to use `luacheck` on NodeMCU code.

I am more of the view: what's wrong with RTFM?  If anyone wants to read up on how to use luacehck then it comes with a lot of good documentation.  May a reference to it in our FAQs.